apps/client/src/App.tsx

```diff
@@ -49,16 +50,18 @@ function App() {
     <ChakraProvider resetCSS theme={theme}>
       <QueryClientProvider client={ontimeQueryClient}>
         <AppContextProvider>
-          <BrowserRouter>
-            <div className='App'>
-              <ErrorBoundary>
-                <TranslationProvider>
-                  <AppRouter />
-                </TranslationProvider>
-              </ErrorBoundary>
-              <ReactQueryDevtools initialIsOpen={false} />
-            </div>
-          </BrowserRouter>
+          <ContextMenuProvider>
```

Comment:
We don't need to wrap the entire app in the context menu.
Wrapping the element with `data-testid:"event-editor"` in `Editor.tsx` would be enough

Although your solution would have generally been correct, in ontime's case, the views are not interactive and often run in devices like TVs. No need to add code there

Reply:
Addressed with https://github.com/cpvalente/ontime/pull/410/commits/c9f0b08e5da9e4f0e1c297758c0c10b805f43740